src/main/java/it/eng/idsa/dataapp/service/MultiPartMessageService.java

```diff
@@ -17,6 +19,12 @@
  * Service Interface for managing MultiPartMessage.
  */
 public interface MultiPartMessageService {
+	//check market4.0-data_app_test_BE/src/main/resources/dataFiles/contract_agreement.json for more information
+	static final String DEFAULT_CONTRACT_AGREEMENT = "https://w3id.org/idsa/autogen/contract/restrict-access-interval";
+	static final String DEFAULT_TARGET_ARTIFACT = "http://w3id.org/engrd/connector/artifact/1";
```

Comment:
Is this one now like this or should be different, related with the changes in MMP?

Reply:
It's like this since this refers to the contract agreement from our policy examples and not the contract offers from the self description.